internal/fnruntime/runner_test.go

```diff
@@ -571,7 +571,7 @@ file:
 		assert.NoError(t, err)
 		assert.NoError(t, populateResourceRef(yml, result))
 
-		out, err := yaml.Marshal(result)
+		out, err := yaml.MarshalWithOptions(result, &yaml.EncoderOptions{SeqIndent: yaml.WideSeqIndent})
```

Comment:
The way this test is written, we shouldn't have to update the `input` part of the tests. And if we used `yaml.Marshal` without options, then `input` and `output` both shouldn't be required to change, right ?

Reply:
Since there are no sequence nodes in the tests we can just use default kyaml marshall. Updated it.